harness/features/variable.local.test.ts

```diff
@@ -0,0 +1,356 @@
+import { 
+    getConnectionStringForProxy, 
+    forEachSDK, 
+    describeIf, 
+    createClient, 
+    createUser, 
+    callVariable, 
+    callOnClientInitialized,
+    forEachVariableType,
+    waitForRequest
+} from '../helpers'
+import { Capabilities, SDKCapabilities } from '../types'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../nock'
+import { config } from '../mockData/index'
+import nock from 'nock'
+
+jest.setTimeout(10000)
+
+const scope = getServerScope()
+
+const expectedVariablesByType = {
+    // these should match the config in mockData
```

Comment:
does this mean like "the tests should prove that these match the config", or is it a note to us that the data defined here needs to match whats in the config? If so can you just use what's in the config or derive from it somehow?